src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarShippingInformation.tsx

```diff
@@ -72,7 +80,7 @@ const ArtworkSidebarShippingInformation: React.FC<
         </Text>
       )}
 
-      {!!shippingInfo && (
+      {!!shippingInfo && !globalArtsyShipping && (
```

Comment:
should we account for the unleash feature flag here so we do not even try to render estimates when feature is not enabled?